### ISSUE TITLE
Loss goes to 0 for image data layer

### ISSUE THREAD

**Issue by awan-10** [BVLC/caffe#4700]
I am trying to run CaffeNet training based on the image_data_layer. I am seeing the following behaviour. The loss value is 7.5 in Iteration 0 and then goes to 0 immediately. Is this expected ?

I0909 20:42:33.842559 136906 solver.cpp:279] Solving CaffeNet
I0909 20:42:33.842563 136906 solver.cpp:280] Learning Rate Policy: step
I0909 20:42:34.010679 136906 solver.cpp:228] Iteration 0, loss = 7.56662
I0909 20:42:34.010716 136906 solver.cpp:244]     Train net output #0: loss = 7.56662 (* 1 = 7.56662 loss)
I0909 20:42:34.010766 136906 sgd_solver.cpp:106] Iteration 0, lr = 0.01
I0909 20:42:34.430747 136906 solver.cpp:228] Iteration 1, loss = 0
I0909 20:42:34.430763 136906 solver.cpp:244]     Train net output #0: loss = 0 (* 1 = 0 loss)
I0909 20:42:34.430768 136906 sgd_solver.cpp:106] Iteration 1, lr = 0.01
I0909 20:42:34.875175 136906 solver.cpp:228] Iteration 2, loss = 0
I0909 20:42:34.875191 136906 solver.cpp:244]     Train net output #0: loss = 0 (* 1 = 0 loss)
I0909 20:42:34.875196 136906 sgd_solver.cpp:106] Iteration 2, lr = 0.01
I0909 20:42:34.875416 136906 blocking_queue.cpp:50] Data layer prefetch queue empty
I0909 20:42:35.293489 136906 solver.cpp:228] Iteration 3, loss = 0
I0909 20:42:35.293507 136906 solver.cpp:244]     Train net output #0: loss = 0 (* 1 = 0 loss)
I0909 20:42:35.293512 136906 sgd_solver.cpp:106] Iteration 3, lr = 0.01
I0909 20:42:35.796902 136906 solver.cpp:228] Iteration 4, loss = 0
I0909 20:42:35.796919 136906 solver.cpp:244]     Train net output #0: loss = 0 (* 1 = 0 loss)
I0909 20:42:35.796924 136906 sgd_solver.cpp:106] Iteration 4, lr = 0.01
I0909 20:42:36.378119 136906 solver.cpp:228] Iteration 5, loss = 0

The head of my prototxt file is:

name: "CaffeNet"
layer {
  name: "data"
  type: "ImageData"
  top: "data"
  top: "label"
  include {
    phase: TRAIN
  }
  transform_param {
    mirror: true
    crop_size: 227
    mean_file: "/users/xyz/projects/caffe/nvcaffe/caffe/data/ilsvrc12/imagenet_mean.binaryproto"
  }

  image_data_param {
    source: "/users/xyz/projects/caffe/nvcaffe/caffe/train.txt"
    batch_size: 128
    new_height: 256
    new_width: 256
  }
}
layer {
  name: "data"
  type: "Data"
  top: "data"
  top: "label"
  include {
    phase: TEST
  }
  transform_param {
    mirror: false
    crop_size: 227
    mean_file: "/users/xyz/projects/caffe/nvcaffe/caffe/data/ilsvrc12/imagenet_mean.binaryproto"
  }
  data_param {
    source: "/lus/scratch/xyz/imagenet-on-lustre/examples/imagenet/ilsvrc12_val_lmdb"
    batch_size: 50
    backend: LMDB
  }
}

**Comment by shelhamer** [comment]
Please ask usage questions on the mailing list. (Loss of 0 is suspicious, and usually indicates some kind of data issue or overfitting.)

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
